### PR TITLE
Add Parameter for all contract settable variable

### PR DESCRIPTION
**Pull request by nz-neooo**

### CODE REVIEW

File: contracts/interfaces/IParameter.sol
Comment (ianhe8x) [inline contracts/interfaces/IParameter.sol:1]: This is more a lib than interface, can you move it under utils?

Comment (nz-neooo) [reply contracts/interfaces/IParameter.sol:1]: It will works, but this event will be not included in ABI

Comment (ianhe8x) [reply contracts/interfaces/IParameter.sol:1]: leave it abstract then, it is not an interface anyway.

Comment (nz-neooo) [reply contracts/interfaces/IParameter.sol:1]: Ok, moved

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**